src/fam.rs

```diff
@@ -5,6 +5,18 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the THIRD-PARTY file.
```

Comment:
Could you also improve the license file name here? Thanks!

Reply:
Sure. Done.